### ISSUE TITLE
Website update: general

### ISSUE THREAD

**Issue by echelleburns** [DanOvando/marlin#79]
I think it would be useful to provide some figures to show what the habitats going into the model look like spatially, so that users can see how that directly affects the model outcome without having to do some digging on their own

**Comment by DanOvando** [comment]
Good idea, might be nice to pretend to load a local csv to make clearer that you don't have to simulate the habitat yourself like we usually do in here
